### PR TITLE
[TVMLKit] Add TVMLKit binding for xcode11.2b2

### PR DESCRIPTION
**Pull request by wachaudh**
Add TVMLKit binding for xcode11.2b2

### CODE REVIEW

File: src/tvmlkit.cs [hunk 1]
```diff
@@ -120,6 +120,7 @@ public enum TVImageType : long {
 	[ErrorDomain ("TVMLKitErrorDomain")]
 	public enum TVMLKitError : long {
 		Unknown = 1,
+		InvalidArguments,
```
Comment (mandel-macaque) [inline src/tvmlkit.cs]: The order of the enums matter. This changes makes InternetUnavailable to be the old one +1. Which is bad it the users do depend on the casting of enums to long/

**Comment by wachaudh** [comment]
maccore issue: https://github.com/xamarin/maccore/issues/2039

File: src/tvmlkit.cs (hunk 1 above)
Comment (Therzok) [reply src/tvmlkit.cs]: It's basically an ABI break.

Comment (Therzok) [reply src/tvmlkit.cs]: Not just enum to long. It breaks interop too.

**Pull request closed without merging**